test/utils/initMatchers.ts

```diff
@@ -5,6 +5,61 @@ import { prettyDOM } from '@testing-library/react/pure';
 import { computeAccessibleName } from 'dom-accessibility-api';
 import formatUtil from 'format-util';
 
+chai.use(chaiDom);
+
+// https://stackoverflow.com/a/46755166/3406963
+declare global {
+  namespace Chai {
+    interface Assertion {
+      /**
+       * checks if the element in question is considered aria-hidden
+       * Does not replace accessibility check as that requires display/visibility/layout
+       * @deprecated Use `inaccessible` + `visible` instead
+       */
+      toBeAriaHidden(): void;
+      /**
+       * Check if an element is not visually hidden
+       */
+      toBeVisible(): void;
+      /**
+       * checks if the element is inaccessible
+       */
+      toBeInaccessible(): void;
+      /**
+       * checks if the accessible name computation (according to `accname` spec)
+       * matches the expectation.
+       * @see https://www.w3.org/TR/accname-1.2/
+       * @param name
+       */
+      toHaveAccessibleName(name: string): void;
+      /**
+       * checks if the element is focused
```

Comment:
If we want to care about the format of the JSDoc for our internal helpers:

```suggestion
       * Checks if the element is focused.
```

If we don't want to care, we can ignore it :)

Reply:
I'll just rewrite them. The current descriptions mostly say nothing.

Reply:
done